### PR TITLE
RAC-6214: add nfs operation for delete volume.

### PR DESCRIPTION
**Pull request by yaolingling**
**Background**
     The PR is used to add nfs file operations for delete volume. The story: https://rackhd.atlassian.net/browse/RAC-6214 has two PRs. The first PR only has cifs file operation. Details in : 
PR 1 
https://github.com/RackHD/on-taskgraph/pull/334
https://github.com/RackHD/on-tasks/pull/558

**Reviewers**
@nortonluo @AlaricChan @anhou @lanchongyizu

### CODE REVIEW

**Comment by yaolingling** [comment]
test this please

**Comment by anhou** [comment]
please fix ''stderr' is defined but never used.'

File: lib/jobs/dell-wsman-delete-volume-updateXml.js [hunk 1]
```diff
@@ -88,5 +83,26 @@ function WsmanDeleteVolumeXmlFactory(
         });
     };
 
+    WsmanDeleteVolumeXmlJob.prototype.updateXml = function(fileData, volumeId){
+        var xmlFile = String.fromCharCode.apply(null, new Uint16Array(fileData));
+        var doc = new xmldom().parseFromString(xmlFile, 'application/xml');
+        var components = doc.getElementsByTagName('Component');
+        var deleteIndex = [];
+        for(var i = 0; i < components.length; i++){       //jshint ignore:line
```
Comment (mcgG) [inline lib/jobs/dell-wsman-delete-volume-updateXml.js]: Why not use `for (var i in components)` i here represents for index

File: lib/utils/job-utils/nfs-client.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    if(mkdirError){
+                        throw mkdirError;
+                    }
+                    logger.debug('sudo mkdir -p ' + self.mountDir + ' successfully.');
+                }
+            );
+        }).then(function(){
+            childProcess.exec(
+                'mount ' + self.shareIP + ':' + self.shareName + ' ' + self.mountDir,
+                function(mountError, stdout, stderr){
```
[the remark this answers is not in the document]
Comment (nortonluo) [reply lib/utils/job-utils/nfs-client.js]: Use // jshint ignore:line to ignore this hound warning.

**Pull request merged**